lib/launchers/openstack10.rb

```diff
@@ -78,6 +80,11 @@ def initialize(**options)
       self.get_token()
     end
 
+
+    def proxy_set?()
+      return @proxy ? true : false
+    end
```

Comment:
Please rename method to `proxy?`.

Reply:
I choose the same name with RestClient https://www.rubydoc.info/gems/rest-client/RestClient#proxy_set%3F-class_method

Reply:
any strong reason to use `proxy?` ?

Reply:
Strong reason is avoiding length when it gives no additional information to the reader. `set` in this case tells nothing to help a code reader. And for code writer it is just some suffix to remember.

If you want to clarify the method like `os_api_access_proxy` (I'm not sure this is a very good name but giving a quick example) then fine.

At the end it is not only about correctness but also style the project maintainer tries to establish. I'm not saying what you did is wrong. I'm just asking you to change to a shorter version unless you have an even better idea.

Actually having a method `proxy?` or `proxy_set?` is not needed at all because it adds code length that is not really necessary to achieve end goal. See my other comment.

Reply:
Thanks